x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -441,7 +441,12 @@ public enum Cap {
         /**
          * Support simplified syntax for named parameters for field and function names.
          */
-        NAMED_PARAMETER_FOR_FIELD_AND_FUNCTION_NAMES_SIMPLIFIED_SYNTAX(Build.current().isSnapshot());
+        NAMED_PARAMETER_FOR_FIELD_AND_FUNCTION_NAMES_SIMPLIFIED_SYNTAX(Build.current().isSnapshot()),
+
+        /**
+         * Fix pushdown of LIMIT past MV_EXPAND
+         */
+        FIX_MV_EXPAND_LIMIT_PUSHDOWN;
```

Comment:
Nit: Unless this PR fixes all issues around limit and mvExpand, add a PARTIAL_ prefix or describe what the PR does - ADD_LIMIT_INSIDE_MV_EXPAND.

Reply:
I think it fully addresses the LIMIT pushdown problem, but I think the name you are suggesting is more descriptive, I'll change it anyway